misk/src/main/kotlin/misk/web/exceptions/ExceptionHandlingInterceptor.kt

```diff
@@ -47,11 +47,14 @@ class ExceptionHandlingInterceptor(
     try {
       chain.proceed(chain.httpCall)
     } catch (th: Throwable) {
-      val response = toResponse(th)
       try {
         if (chain.httpCall.dispatchMechanism == DispatchMechanism.GRPC) {
+          // This response object is only used for determining the status code. toGrpcResponse
+          // will provide a more useful log instead.
+          val response = toResponse(th, suppressLog = true)
```

Comment:
the `GrpcErrorResponse` also has a status code. I wonder if we should use that instead so we avoid the `suppressLog` param.

Reply:
It's a gRPC status code, so we'll need to convert it back to an HTTP response code. I'd rather not go through multiple layers of translation (since it was originally derived from an HTTP status code), due to the risk of losing specificity.